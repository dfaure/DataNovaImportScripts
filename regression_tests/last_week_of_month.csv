06173A;BOISSERON AP;2020-10-27;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-27;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-30;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-02;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-17;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-19;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-06;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-26;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-04;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-25;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-28;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-29;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-18;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-07;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-10;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-13;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-02;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-24;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-11;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-16;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-20;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-23;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-14;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-01;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-15;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-10-28;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-10-29;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-10-30;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-12;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-09;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-21;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-22;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-03;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-08;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-05;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-10-26;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-10;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-03;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-08;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-01;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-13;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-11-07;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-11;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-10-31;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-20;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-08;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-07;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-15;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-30;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-09;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-16;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-25;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-05;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-26;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-21;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-21;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-12;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-18;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-04;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-19;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-17;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-03;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-16;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-22;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-23;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-20;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-17;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-19;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-06;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-31;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-15;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-24;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-14;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-29;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-13;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-27;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-01;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-18;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-11;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-28;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-09;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-05;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-14;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-06;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-10;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2020-12-04;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-02;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-12;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-25;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-23;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-26;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-22;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-24;FERME;;;;;;;;;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-27;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
06173A;BOISSERON AP;2021-01-28;08:30-12:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
