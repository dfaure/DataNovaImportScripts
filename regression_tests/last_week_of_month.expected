06173A|BOISSERON AP|Mo-Sa 08:30-12:00; Sa[-1] off
